### PR TITLE
Add "system" to the completion list for plugin "chruby"

### PR DESCRIPTION
**Pull request by franklinyu**
Detect system Ruby in default `PATH`, and provide "system" completion if Ruby is found. Tested with [the Dockerfile](https://gist.github.com/franklinyu/b8deda6a5093a17b575679b5808b371f).

### CODE REVIEW

**Comment by franklinyu** [comment]
@webframp Any suggestion?

**Comment by webframp** [comment]
Seems reasonable to me, I didn't have this particular need but it looks good.

**Comment by franklinyu** [comment]
So no one want to test it out by now?

File: plugins/chruby/chruby.plugin.zsh [hunk 1]
```diff
@@ -95,5 +95,11 @@ function chruby_prompt_info() {
 }
 
 # complete on installed rubies
-_chruby() { compadd $(chruby | tr -d '* ') }
+_chruby() {
+    compadd $(chruby | tr -d '* ')
+    local default_path='/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin'
+    if ( PATH=${default_path} type ruby > /dev/null); then
```
Comment (mcornella) [inline plugins/chruby/chruby.plugin.zsh]: This should be `&>/dev/null` to redirect also stderr. You can also get rid of the parentheses.

**Comment by franklinyu** [comment]
@mcornella Any more suggestions?

**Comment by mcornella** [comment]
That's great, thanks for your contribution!

**Pull request merged**